test/cmdlineTests/transient_storage_layout_value_types_interleaved_with_storage/output.json

```diff
@@ -0,0 +1,24 @@
+{
+    "errors": [
+        {
+            "component": "general",
+            "errorCode": "2314",
+            "formattedMessage": "ParserError: Expected identifier but got 'external'
+ --> fileA:8:13:
+  |
+8 |     address external transient addr;
+  |             ^^^^^^^^
+
+",
+            "message": "Expected identifier but got 'external'",
```

Comment:
The test doesn't compile. I guess `external` is not allowed on storage variables?

Reply:
Yeah, it is not allowed. I should have tested before and then actually looked at the result.
Fixed.